test/fixtures/vpc_sc_project/main.tf

```diff
@@ -14,28 +14,12 @@
  * limitations under the License.
  */
 
-provider "google" {
-  version = "~> 3.30"
-}
-
-provider "google-beta" {
-  version = "~> 3.30"
-}
-
-provider "null" {
-  version = "~> 2.1"
-}
-
-provider "random" {
-  version = "~> 2.2"
-}
-
 locals {
   perimeter_name = "regular_service_perimeter_${var.random_string_for_testing}"
 }
 
 module "regular_service_perimeter_1" {
-  source         = "terraform-google-modules/vpc-service-controls/google//modules/regular_service_perimeter"
+  source         = "github.com/terraform-google-modules/terraform-google-vpc-service-controls//modules/regular_service_perimeter"
```

Comment:
Can we open an issue to track revert of this in the future?

Reply:
You want me to do this before PR is merged?